scrapy/utils/project.py

```diff
@@ -44,7 +46,7 @@ def project_data_dir(project: str = "default") -> str:
     return str(d)
 
 
-def data_path(path: str, createdir: bool = False) -> str:
+def data_path(path: Union[str, PathLike, Path], createdir: bool = False) -> str:
```

Comment:
`pathlib.Path` is a subclass of `os.PathLike` so it's not needed to be mentioned separately here.

Reply:
@wRAR updated

Reply:
I meant removing `Path`, not removing `PathLike`. `Union[str, PathLike]` is the usual hint for these APIs.

Reply:
Oh, right. Updated.